CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.4.0 - 2018-09-18
+
+* Implement Value for 128 bit integers
+* Bump miminum supported Rust version to 1.26
+
```

Comment:
Should we add a note to add support Rust2018-style macro imports fixed in #183?

Reply:
@ubnt-intrepid Yes. :)

Reply:
@fintelia could you add the note to CHANGELOG.md?

Reply:
Done